<launch>
  <node pkg="ieee2016_vision" type="camera_manager.py" name="camera_manager" output="screen">
    <param name="cam_1_index" value="0"/>
    <param name="cam_1_width" value="1280"/>
    <param name="cam_1_heigth" value="720"/>

    <param name="cam_2_index" value="1"/>
    <param name="cam_2_width" value="640"/>
    <param name="cam_2_heigth" value="480"/>
  </node>
</launch>
